decidim-debates/lib/decidim/debates/engine.rb

```diff
@@ -45,7 +45,7 @@ class Engine < ::Rails::Engine
           badge.reset = lambda do |user|
             debates = Decidim::Comments::Comment.where(
               author: user,
-              decidim_root_commentable_type: "Decidim::Debates::Debate"\
+              decidim_root_commentable_type: "Decidim::Debates::Debate" \
```

Comment:
Do we really need the `\` ? 
```suggestion
              decidim_root_commentable_type: "Decidim::Debates::Debate"
```

Reply:
I don't think so. Committed your suggestion at 09a553e369ff1e78cf550aa4de5cd4a669b69b7c.